spec/support/devolved_nations_helper.rb

```diff
@@ -0,0 +1,10 @@
+module DevolvedNationsHelpers
+  def has_devolved_nations_component(text, nations = nil)
```

Comment:
You could write this as a full matcher, allowing you to do stuff in your tests like:

`expect(page).to have_devolved_nations_component("Applies to England")`